azurerm/internal/services/policy/tests/data_source_policy_definition_test.go

```diff
@@ -104,7 +152,7 @@ resource "azurerm_policy_definition" "test_policy" {
   name         = "acctestpol-%d"
   policy_type  = "Custom"
   mode         = "All"
-  display_name = "acctestpol-%d"
+  display_name = "acctestpol-display-%d"
```

Comment:
Could we get some uppercase letters here?

Reply:
DOne